app/models/role.rb

```diff
@@ -10,22 +10,26 @@ class Role < ActiveRecord::Base
   scope :admins, -> { where(name: "admin") }
   scope :moderators, -> { where(name: %w(moderator admin)) }
 
-  def self.is_admin?(person)
-    exists?(person_id: person.id, name: "admin")
+  def self.admin?(person_id)
+    exists?(person_id: person_id, name: "admin")
```

Comment:
On second thought, maybe they should handle both for now, `id = person.is_a?(Person) ? person.id : person`